Group 2 of Group 8 Review on 20200325

skjohansen opened the pull request.

koffe-bot committed:
Add files via upload

koffe-bot committed:
Delete TheSpaceport.sln

koffe-bot committed:
Add files via upload

koffe-bot committed:
Add files via upload

albin1997536 reviewed (commented):
Jag **hittade** bara en CS fil. Den var auto genererad (_"// Generated by the MSBuild WriteCodeFragment class."_) så kan inte säga så mycket.

Adgnascor commented:
> Jag **hittade** bara en CS fil. Den var auto genererad (_"// Generated by the MSBuild WriteCodeFragment class."_) så kan inte säga så mycket.

Vi fick också den först, har blivit något fel. Skriv till Stephan så fixar han korrekt till er :)

albin1997536 commented:
> > Jag **hittade** bara en CS fil. Den var auto genererad (_"// Generated by the MSBuild WriteCodeFragment class."_) så kan inte säga så mycket.
> 
> Vi fick också den först, har blivit något fel. Skriv till Stephan så fixar han korrekt till er :)

Tack, skall göra det.

Commit:
Backup for major changes

Commit:
-

Commit:
cleanup

Commit:
Update Program.cs

Commit:
Garage now has a limit

Commit:
Fixed signed variable rolling over

TODO:

⁌Log docking event to DB
⁌Replace randomized characters with user input

Commit:
none cached loading

Commit:
fixed a bug where randoms could impersonate Luke

Commit:
Fixed a bug that messed up the consoles colors

Commit:
Minor Changes and some notes for after DB is created.

Commit:
Added visual representation for parking deck capacity, Updated member variabels names

Commit:
Merge branch 'master' of https://github.com/PGBSNH19/the-spaceport-grupp-8

Commit:
Merge Conflict

added progress bar,
updated Platform class

Commit:
fixed merge conflict

Commit:
Re-entering things that were removed

koffe-bot committed:
Add files via upload

koffe-bot committed:
Delete Coder.rar

koffe-bot committed:
Add files via upload

Commit:
Ships are no longer player selected. Theyre now randomized to support async

Ships are no longer player selected. Theyre now randomized to support async API calls

Commit:
Revert "Ships are no longer player selected. Theyre now randomized to support async"

This reverts commit 75a83f7b5160ae38b0037aceb8f85c6a53089943.

Commit:
Changed Logic for Tax

Commit:
Merge branch 'master' of https://github.com/PGBSNH19/the-spaceport-grupp-8

Commit:
Merge branch 'master' of https://github.com/PGBSNH19/the-spaceport-grupp-8

Commit:
Revert "Merge branch 'master' of https://github.com/PGBSNH19/the-spaceport-grupp-8"

This reverts commit 7c5e5b5b8172658559ddae5b7e22a81ecf53b713, reversing
changes made to 33d5b9e533b78722c3699ecd4f713133d48c8f09.

Commit:
API calls are now all handled by a single asynchronous method and the program can now handle multiple parking decks . Added WiP ship departure method (requieres SQL integration to be completed)

koffe-bot committed:
Add files via upload

koffe-bot committed:
Add files via upload

koffe-bot committed:
Add files via upload

koffe-bot committed:
Add files via upload

Commit:
new project now uses Async without exception

TODO:
pre-load ships instead of chars due to ship quantity?

koffe-bot committed:
Add files via upload

Commit:
Due to ship quantity, they're now prioritized to load async

Commit:
Minor Tweeks and Added the math for 5 currency/30 min

Adgnascor reviewed (commented):
s

Commit:
Minor Tweekz
